addons/mail/models/mail_message.py

```diff
@@ -1142,9 +1143,7 @@ def _message_notification_format(self):
             'notifications': message.notification_ids._filtered_for_web_client()._notification_format(),
         } for message in self]
 
-    def _notify_message_notification_update(self):
-        """Send bus notifications to update status of notifications in the web
-        client. Purpose is to send the updated status per author."""
+    def _filter_can_read_record(self):
```

Comment:
Should be in alphabetical order, ideally

Reply:
I'm not sure of what you mean. What should be in alphabetical order? The method `_filter_can_read_record` in `addons/mail/models/mail_message.py`? Or the filtered messages?